r/extensions.casult

```diff
@@ -0,0 +1,2 @@
+SELECT 1 FROM NON_EXISTING_TABLE;
```

Comment:
Is this file added by mistake?

Reply:
no, just test whether the `extension` param work